common/build/eslint-config-fluid/printed-configs/default.json

```diff
@@ -811,21 +822,15 @@
         "semi": [
             "off"
         ],
-        "semi-spacing": [
-            "error"
-        ],
         "space-before-blocks": [
             "error"
         ],
-        "space-before-function-paren": [
-            "off"
```

Comment:
Maybe space-before-function-paren is covered by the typescript-eslint version?  Should make sure it errors.

Reply:
Yeah, this is superseded by @typescript-eslint/space-before-function-paren. I updated the comments in the config.